### DISCUSSION TITLE
Seconds not visable when exiting the program.

### DISCUSSION THREAD

**Discussion by noahnao** [martinet101/ElevenClock#935] [Q&A]
The Seconds counter is not visable when exiting the program, can you make it so that you can see the seconds regardless if the program is open or not, like replaysing or editing the already existing windows clock. I want this because Elevenclock hogs system resouses whilst gaming or other demabing tasks, and i have a very lowend system, that needs every ounce of preformance to be useable.

Thank you in advance, Noah.

**Comment by marticliment** [comment]
Hi @noahnao, the way in that ElevenClock shows the seconds is by showing over the windows clock, since it can't be customized anymore. So what you are asking won't be possible

If you wish, you can enable the "low-cpu" mode in the settings window, but ElevenClock is not much resource-intensive. 

However, please check ypur task manager for cpu usage, because there have been a few reports of ElevenClock misbehaving and consuming tons of CPU. Does this apply for you?
